TACC/Core-Portal
# Quick: Remove <Message type="info"> icon

## Overview: ##

Remove the icon from the `<Message type="icon">` (and only that `type`), because:
- The design shows such informational messages without an icon.
- Such informational messages are _either_ not using `<Message>` _or_ are using `<Message type="warn">` (both wrong).
- Design has agreed to no icon for info messages.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1615506893072000?thread_ts=1615505431.070500&cid=CQUA4D5KJ)

## Related Jira tickets: ##

* (tangentially) [FP-959](https://jira.tacc.utexas.edu/browse/FP-959) (see PR #360)

## Summary of Changes: ##

- __Fix__: Hide the icon for `<Message type="icon">`.

## Testing Steps: ##
1. Load "UI Patterns" section.
2. Scroll to "Message & Notification".
3. Confirm that __neither__ `<InlineMessage>` __nor__ `<SectionMessage>` of type `info` show an icon.
4. Confirm that there is no change to the other permutations' rendering.

## UI Photos:

### Before
![Info Message - Before](https://user-images.githubusercontent.com/62723358/111207725-f8767200-8597-11eb-8a16-11dda6d1a5c7.png)

### After
![Info Message - After](https://user-images.githubusercontent.com/62723358/111207715-f6141800-8597-11eb-8a0e-614c91d85cf4.png)
